drivers/include/net/netdev2_ieee802154.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup drivers_netdev_netdev2
+ * @brief
+ * @{
+ *
+ * @file
+ * @brief   Definitions for netdev2 common IEEE 802.15.4 code
+ *
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+#ifndef NETDEV2_IEEE802154_H_
+#define NETDEV2_IEEE802154_H_
+
+#include "net/ieee802154.h"
+#include "net/gnrc/nettype.h"
```

Comment:
No in this file it is needed. For the `proto` member, that is only available when GNRC is present.